src/flow_solver/flow_solver.cpp

```diff
@@ -500,19 +505,23 @@ int FlowSolver<dim,nstate>::run() const
             }
 
             // update time step in flow_solver_case
-            flow_solver_case->set_time_step(time_step);
```

Comment:
If it doesn't impact other cases, we should probably remove the comment before it that says "update time step in flow_solver_case" since that is not being done anymore.

Reply:
I added this line back in case other cases are impacted.

Reply:
Sounds good!